packages/preview-middleware-client/src/flp/initConnectors.ts

```diff
@@ -6,14 +8,16 @@
  * This setup allows for flexibility in using different connectors based on the UI5 version.
  *
  * @example
- * intiConnectors(); // Simply call the function without any arguments.
+ * initConnectors(); // Simply call the function without any arguments.
  * @returns {void}
```

Comment:
According to the changes JSDoc needs to be adjusted to returning `Promise<void>`

Reply:
The `@returns` was part of the JSDoc because it describes the return value of the `initConnectors` function as such. So it's not part of the `@example` that's correct but it still should be part of the JSDoc of `initConnectors` because it describes the return value of the function (see [JSDOC example](https://jsdoc.app/tags-example)). The only thing that was wrong was the type. Although `initConnectors` has been adjusted in this branch to return a promise it was still `void` according to JSDoc.
At least that how it is done e.g. in the [init.ts](https://github.com/SAP/open-ux-tools/blob/main/packages/preview-middleware-client/src/flp/init.ts#L154) of the middleware client. So for consistency reasons I would stick to it.

Technically the TypeScript type checker can [infer](https://www.typescriptlang.org/docs/handbook/2/functions.html#inference) the return type anyway. So `@returns` as well as `: Promise<void>` could be omitted.